include/dlaf/matrix/distribution.h

```diff
@@ -268,10 +268,10 @@ class Distribution {
   /// whose index is the smallest index larger or equal @p global_tile
   /// and which is stored in current process.
   ///
-  /// @pre 0 <= global_tile < nrTiles().get<rc>()
+  /// @pre 0 <= global_tile <= nrTiles().get<rc>()
   template <Coord rc>
   SizeType nextLocalTileFromGlobalTile(SizeType global_tile) const noexcept {
-    assert(0 <= global_tile && global_tile < global_nr_tiles_.get<rc>());
+    assert(0 <= global_tile && global_tile <= global_nr_tiles_.get<rc>());
```

Comment:
Do you need the equal because it can return an index to a tile "out of borders", right?

Reply:
This function is used to find loop limits, therefeore it should be possible to call it with also the index of the last+1 element.